src/featureFlags.js

```diff
@@ -46,9 +46,6 @@ export class FeatureFlags extends events.EventEmitter {
   * Uses the Hathor Unleash Server and Proxy to determine if the
   * wallet should use the WalletService facade or the old facade
   *
-  * @params {string} userId An user identifier (e.g. the firstAddress)
```

Comment:
:bike: The `userId` param docstring should remains.

Reply:
I think it was wrong before, the method does not have any parameters

Reply:
Exactly